images/payload-testing-ui/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:8
+FROM quay.io/centos/centos:stream8
```

Comment:
I'm changing this because with vanilla `centos:8` we are hitting the docker hub rate limit